release-team/role-handbooks/release-notes/README.md

```diff
@@ -2,32 +2,21 @@
-This role is likely to find that work during the first several weeks of the release cycle is very laid back with the bulk of the tasks being
-completed at the end, once the release is firmed up.
+The Release Notes role is responsible for collecting and fine-tuning release-notes from the many contributions to Kubernetes between release cycles. This role is likely to find that work during the first several weeks of the release cycle is very laid back with the bulk of the tasks being completed at the end, once the release is firmed up.
 
 ## Skills and Experience Required:
 
 * Strong written and verbal communications skills
-
 * A working knowledge of Kubernetes concepts
-
 * Project management experience is helpful
 
 ## Tasks
 
-- The Release notes lead and shadows attend burn down meetings, sig-release meetings and follow the sig-release
-slack chat for relevant information throughout the release cycle.
-- One member of the release notes team should be responsible for setting up and running the [release notes tool](https://github.com/marpaia/release-notes)
-to collect generated drafts of all release-notes identified in the current release -- improvements to the tool are
-always welcome!
-- Drafts produced by the tool are copied to a Google Document that gives editing privleges to each member of the
-release-notes team (there is currently no automated way to keep the generated release-notes and the google doc
-concurrent. Some tool to automate this process would be a good task for the 1.13 release team)
-- Closer to the release date, major themes are collected from the release notes draft and the release notes team
-creates a draft in markdown of the identified major themes to send to sig-leads for feedback/confirmation
-- If gentle nudging of sig leads is not effective in retrieving feedback/confirmation, the release-notes team can
-use a reasonable amount of creative liberity in completing the notes
+- The Release notes lead and shadows attend burn down meetings, SIG Release meetings and follow the [#sig-release](https://kubernetes.slack.com/messages/C2C40FMNF) Slack channel for relevant information throughout the release cycle.
+- One member of the release notes team should be responsible for setting up and running the [release notes tool](https://github.com/marpaia/release-notes) to collect generated drafts of all release-notes identified in the current release -- improvements to the tool are always welcome!
+- Drafts produced by the tool are copied to a Google Document that gives editing privleges to each member of the release-notes team (there is currently no automated way to keep the generated release-notes and the google doc concurrent. Some tool to automate this process would be a good task for the 1.13 release team)
+- Closer to the release date, major themes are collected from the release notes draft and the release notes team creates a draft in markdown of the identified major themes to send to sig-leads for feedback/confirmation
```

Comment:
Is it safe to say we want to send the initial draft and release themes to SIG leads for review during Code slush or immediately after Code freeze begins.